payjoin/src/send/mod.rs

```diff
@@ -1200,9 +1196,7 @@ mod test {
     }
 
     #[test]
-    fn handle_known_errors() {
-        use crate::send::error::{ResponseError, WellKnownError};
-
+    fn handle_json_errors() {
```

Comment:
imo known makes more sense here as we are using it everywhere else in the code..

Reply:
this test handles both known and validation errors. If it were split between separate unit tests and only tested known errors i would agree.